cmd/deploy/deploy.go

```diff
@@ -288,7 +288,7 @@ $ okteto deploy --no-build=true`,
 	cmd.Flags().BoolVarP(&options.RunWithoutBash, "no-bash", "", false, "execute the command using the container's default shell instead of bash")
 	cmd.Flags().BoolVarP(&options.RunInRemote, "remote", "", false, "run the deploy commands using Remote Execution")
 
-	cmd.Flags().BoolVarP(&options.Wait, "wait", "w", true, "wait until the deployment finishes")
+	cmd.Flags().BoolVarP(&options.Wait, "wait", "w", false, "wait until the deployment finishes")
```

Comment:
@andreafalzetti could you use the same description that our docs? same for the `timeout` flag:
https://github.com/okteto/docs/pull/835
Also, your backport might be easier if you also backport this one:
https://github.com/okteto/okteto/pull/4453